rules/library.bzl

```diff
@@ -615,6 +616,9 @@ def apple_library(
             )
 
             swift_sources.append(intent_name)
+
+            # Refresh `has_swift_sources` since `swift_sources` has changed
+            has_swift_sources = len(swift_sources) > 0
```

Comment:
I dont think this could ever change right? Since to be in this `if` we must have had swift_sources and so adding a swift source file wont ever make this false.

Reply:
Oh good catch, I'll remove it

Reply:
Force pushed with the change